README.md

```diff
@@ -0,0 +1,12 @@
+# [Madamiru](https://github.com/mtkennerly/madamiru)
+
+## Adding drives
```

Comment:
no user will ever see this.

Reply:
That's fair. I've removed it and just left a link to the original project :)

Reply:
I was just setting expectations. This kind of info can be useful for maintainers.  (and in that case explaining the existing permission instead)